### PR TITLE
Update build-instructions-windows.md

### PR DESCRIPTION
**Pull request by FranzDeCopenhague**
This is my first time building electron and it was my first issue following the documentation.

### CODE REVIEW

File: docs/development/build-instructions-windows.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ there is no Visual Studio project generated.
 $ cd electron
 $ python script\bootstrap.py -v
 ```
+To avoid errors running the bootstrap script, run the bootstrap script as administrator
```
Comment (ckerr) [inline docs/development/build-instructions-windows.md]: I don't think this is correct; Electron doesn't require the bootstrap script to be run as administrator AFAIK.

What errors did you encounter that required admin permissions?

Comment (FranzDeCopenhague) [reply docs/development/build-instructions-windows.md]: Sorry, I don't remember the error but I need it to finalize the bootstrap. Also, I did try to clone electron in a different folder and now I am able to complete the bootstrap without administrator rights. My guess, at first time, the script tried to do something that requires running as administrator and now it is not required.

**Comment by felixrieseberg** [comment]
Hey! With you there, Administrator rights are for sure not required. @FranzDeCopenhague we should figure out what your machine is missing and document that instead.

**Comment by FranzDeCopenhague** [comment]
Definitely was something on my machine, but now is not reproducible. I can drop and close this pull, or instead, we can add a troubleshooting note in case of bootstrap fails, running as an administrator may help.

**Comment by ckerr** [comment]
Running dev tools as admin is a general security risk. I'd prefer that we find out what the issue was and then decide how to address it.

I'm going to close this PR on those grounds, but please do keep communication open: if you encounter this problem again, please open an issue on it. Thank you!

**Pull request closed without merging**